_source/logzio_collections/_security-sources/aws-security-hub.md

```diff
@@ -88,4 +108,101 @@ Give the stack some time to deploy and the resources to get created. Once this i
 If you still don't see your events, see [log shipping troubleshooting]({{site.baseurl}}/user-guide/log-shipping/log-shipping-troubleshooting.html).
 
 </div>
+
+</div>
+</div>
+  
+<!-- tab:end -->
+
+<!-- tab:start -->
+<div id="manual-cloudformation-deployment">
+
+#### Manual CloudFormation deployment
+
+{% include log-shipping/note-lambda-test.md %}
+
+**Before you begin, you'll need**:
+
+* A CloudFormation stack
+* An S3 bucket to store the CloudFormation package
+
+<div class="tasklist">
+
+##### Select the Logz.io AWS Security Hub extension
+
+1. Navigate to **CloudFormation > Registry > Public extensions**.
+2. Set **Extension type > Modules** and **Publisher > Third party**.
+3. Select **logzio::awsSecurityHub::collector::MODULE**.
+
+
+##### Activate the Logz.io Kinesis extension
+
+1. On the **logzio::awsSecurityHub::collector::MODULE** select **Activate**.
+2. In the **Extension details** section, select **Use default**.
+3. In the **Automatic updates** section, select **On**.
+4. Select **Activate extension**.
+
+##### Copy the configuration template
+
+On the **logzio::awsSecurityHub::collector::MODULE** page, navigate to **Example template** and select **Copy template**.
+
+##### Add your stack values to the configuration template
+
+```yaml
+{
+    "Resources": {
+        "MyModule": {
+            "Type": "logzio::awsSecurityHub::collector::MODULE",
+            "Properties": {
+                "logzioOperationsToken": "logzioOperationsToken",
+                "logzioListener": "logzioListener",
+                "logzioLogLevel": "logzioLogLevel"
+            }
+        }
+    }
+}
+```
+
+Save the template as a yaml file and add the values of your stack to the as per the table below.
+
+| Parameter | Description |  
+|---|---|---|
+| logzioOperationsToken | Your Logz.io account token. {% include log-shipping/log-shipping-token.html %} |
+| logzioListener | {% include log-shipping/listener-var.html %} | 
+| logzioLogLevel | Required log level.  | 
```

Comment:
I think it's worth mentioning the possible log levels that this integration allows, they are mentioned in the "automated deployment" tab